### PR TITLE
fix: Add samuelbartels20

### LINKED ISSUE #898: Samuel Bartels
## Profile Screenshot
Screenshot 2021-12-09 at 23 50 44

### PR DESCRIPTION
**Pull request by samuelbartels20**
resolves #898 

## Checklist for profile addition

- [x] Name the PR `fix: Add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory. 
- [x] You've linked the PR to the correct issue.

### CODE REVIEW

File: profiles/samuelbartels20.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/40799981/145493827-298abbdd-1ad2-482a-aad0-f999f50e1de9.png",
+    "issueId": 898,
+    "name": "Samuel Bartels",
+    "username": "samuelbartels20"
+  }
```
Comment (Panquesito7) [inline profiles/samuelbartels20.json:6]: Missing an endline.
```suggestion
}

```

**Review by Panquesito7 — APPROVED** [review]
Great profile, by the way! 🚀

File: profiles/samuelbartels20.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/40799981/145493827-298abbdd-1ad2-482a-aad0-f999f50e1de9.png",
+    "issueId": 898,
+    "name": "Samuel Bartels",
+    "username": "samuelbartels20"
+}
```
Comment (Panquesito7) [inline profiles/samuelbartels20.json:6]:
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by schmelto — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**